FWCore/src/components/PileupTrackHitsMergeTool.cpp

```diff
@@ -0,0 +1,89 @@
+#include "PileupTrackHitsMergeTool.h"
```

Comment:
Ah yes, I didn't realize the possibility of doing it with tools like this. Seems much better, thanks!